src/components/Navigation.vue

```diff
@@ -40,9 +62,11 @@
   </nav>
 </template>
 
-<script>
+<script lang="ts">
+import { defineComponent } from 'vue';
+
 /* eslint-disable max-len,vue/max-len */
```

Comment:
is this supress still needed?

Reply:
Does not seem like it's still needed. Which is a bit surprising to me? Since some of those lines are veeery long...

Reply:
maybe make sure the check still works. It should trigger after 120 characters if I'm not mistaken (we configured it somewhere)

Reply:
Had to move some of our rules from `package.json` to `.eslintrc.js`